src/components/views/beta/BetaCard.tsx

```diff
@@ -36,17 +36,27 @@ interface IProps {
     featureId: string;
 }
 
-export const BetaPill = ({ onClick }: { onClick?: () => void }) => {
+interface IBetaPillProps {
+    onClick?: () => void;
+    tooltipTitle?: string;
+    tooltipCaption?: string;
+}
+
+export const BetaPill = ({
+    onClick,
+    tooltipTitle = _t("This is a beta feature"),
+    tooltipCaption = _t("Click for more info"),
```

Comment:
these are only shown when `onClick` is passed, would be better if we specified the props using an XOR to demonstrate this